charts/fluentd/values.yaml

```diff
@@ -10,7 +10,16 @@ kind: "DaemonSet"
 image:
   repository: "fluent/fluentd-kubernetes-daemonset"
   pullPolicy: "IfNotPresent"
-  tag: ""
+  tag: ""  # Set to "-" to not use the default value
```

Comment:
Is it possible to not require "-" in tag? You can use the helper functions to conditionally do this.

Also remember that a hash will always overrite the tag, even if hash and tag do not match

'image:tag@hash' will overwrite 'image:tag'

After that remove comment in front plz